src/shared/utils/index.ts

```diff
@@ -539,3 +539,45 @@ export const forceAsArray = <T>(objectOrArray: T | T[] | null | undefined) => {
       : [objectOrArray]
     : [];
 };
+
+/**
+ * Checks if a string is a valid URL.
+ * @param {string} url
+ */
+export const isURL = (url: string) => {
+  try {
+    new URL(url);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+/**
+ * Converts delta url to a fusion url
+ * @param url
+ * @returns
+ */
+
+export const deltaUrlToFusionUrl = (url: string, nexusWebBase: string) => {
+  const projectUrlPattern = /projects\/([\w-]+)\/([\w-]+)\/?$/;
```

Comment:
How about using named regex groups here to make easier to interpret url param being extracted? Like this:

  const projectUrlPattern = /projects\/(?<org>[\w-]+)\/(?<project>[\w-]+)\/?$/;
  const resourceUrlPattern = /resources\/(?<org>.[^/]*)\/(?<project>.[^/]*)\/(.[^/]*)\/(.*)/;
  const fileUrlPattern = /files(?<orgprojectpath>\/([\w-]+)\/([\w-]+))/;

and then refer to groups like this projectUrl.groups?.org when constructing Fusion url to return. So like this:

  if (projectUrl) {
    return `${nexusWebBase}/admin/${projectUrl.groups?.org}/${projectUrl.groups?.project}`;
  }
  if (resourceUrl) {
    return `${nexusWebBase}/admin/${resourceUrl.groups?.org}/${
      resourceUrl.groups?.project
    }/${encodeURIComponent(url)}`;
  }
  if (fileUrl) {
    return `${nexusWebBase}/admin${
      fileUrl.groups?.orgprojectpath
    }/${encodeURIComponent(url)}`;
  }